mapbox_tilesets/scripts/cli.py

```diff
@@ -232,12 +232,19 @@ def status(tileset, token=None, indent=None):
     mapbox_api = utils._get_api()
     mapbox_token = utils._get_token(token)
     s = utils._get_session()
-    url = "{0}/tilesets/v1/{1}/status?access_token={2}".format(
+    url = "{0}/tilesets/v1/{1}/jobs?limit=1&access_token={2}".format(
         mapbox_api, tileset, mapbox_token
     )
     r = s.get(url)
 
-    click.echo(json.dumps(r.json(), indent=indent))
+    status = {}
+    for job in r.json():
+        status["id"] = job["tilesetId"]
+        status["latest_job"] = job["id"]
+        status["status"] = job["stage"]
+        status["last_modified"] = job["created_nice"]
```

Comment:
I don't think we should frame the time the most recent job was created as the time the tileset was most recently modified: if the most recent job failed, then the tileset was not modified at all. We should try to reinforce the difference between tilesets and jobs.

What do you think about changing the status command to use the jobs endpoint but keeping the output exactly the same, and then documenting that the jobs command is the best way to get detailed info about the most recent job?

Reply:
I like that, I'll make changes to the docs.